SettingsList.py

```diff
@@ -2787,19 +2788,21 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             'On LACS: Vanilla': Shadow and Spirit Medallions.
             'On LACS: Medallions': A configurable amount of Medallions.
             'On LACS: Stones': A configurable amount of Spiritual Stones.
-            'On LACS: Dungeons': A configurable amount of Dungeon Rewards
+            'On LACS: Dungeons': A configurable amount of Dungeon Rewards.
+            'On LACS: Tokens': A configurable amount of Gold Skulltula Tokens.
                                  (Spiritual Stones & Medallions).
```

Comment:
```suggestion
            'On LACS: Dungeons': A configurable amount of Dungeon Rewards
                                 (Spiritual Stones & Medallions).
            'On LACS: Tokens': A configurable amount of Gold Skulltula Tokens.
```

Reply:
Not sure how I missed that second line when adding in tokens. Whoops.